types/es-abstract/2015/ObjectCreate.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function ObjectCreate(proto: object | null, internalSlotsList?: readonly []): any;
```

Comment:
```suggestion
declare function ObjectCreate(proto: object | null, internalSlotsList?: readonly []): object extends proto;
```
?

Reply:
That’s broken syntax.

---

This currently returns `any` to mirror `Object.create`.

Reply:
Just because TS's builtin types are subpar doesn't mean this library's should be. at the very least, `object` seems like a better choice than `any`.